### PR TITLE
feat: add automatically layer publish

### PR DESCRIPTION
**Pull request by iwaduarte**
Sorry, I took too long. I had a very busy last few weeks.

Adding automatic layer publish. It adds a workflow that :

`layers_dispatch.yaml`

1- Checks daily the repository (it can also be set for weeks, every 3 days via CRON)
2- Check the version to see if the layer was already published
3- Publish the layer to all regions
4- Update the layer_version file to avoid republishing (creates a commit)

`update_readme`

1 - Get all layers ARN and tag version.
2 - Updates the readme with the proper ARN and VERSION of the chrome
3 - Depends on the layers_dispatch finished successfully

### CODE REVIEW

**Comment by vladgolubev** [comment]
wow, awesome work, really appreciate it! I will try to set it up a bit later today at most tomorrow!

**Comment by iwaduarte** [comment]
@stefanzweifel Let me know if can help you somehow.

**Comment by stefanzweifel** [comment]
@iwaduarte I'm already more than happy that you prepared this PR for this project. I'm probably going to borrow some of the workflows to generate my own layers for the sidecar-browsershot project.

**Comment by vladgolubev** [comment]
Let's see how it works today at midnight soon!

**Comment by iwaduarte** [comment]
@vladgolubev Nice! If you want, you could test by removing the trigger and adding "on: push" directive.

**Comment by vladgolubev** [comment]
@iwaduarte  awesome! I had to [fix the code a bit](https://github.com/shelfio/chrome-aws-lambda-layer/compare/e1e8defe87ffe2eea3632a8d489718e9a2845138...master) but it works now!! Thanks A TON!!

2023-09-12T21 34 34-65qte6MR@2x

File: .github/workflows/layers_dispatch.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+      - name: Get Previous Tag Version
+        id: prev_tag_version
+        run: echo "::set-output name=tag::$(cat .tag_version || echo '')"
+
+      - name: Get Latest Release URL and Tag Version
+        id: latest_release
+        run: |
+          JSON_RESPONSE=$(curl -s https://api.github.com/repos/Sparticuz/chromium/releases/latest)
+          LATEST_RELEASE_URL=$(echo $JSON_RESPONSE | grep -Po '"zipball_url": "\K[^"]+')
```
Comment (stefanzweifel) [inline .github/workflows/layers_dispatch.yaml:34]: Shouldn't the `LATEST_RELEASE_URL` here point to `assets.0.browser_download_url` @iwaduarte ?
(https://api.github.com/repos/Sparticuz/chromium/releases/latest)

The unziped file downloaded through the `zipball_url` contains the _raw_ files of the [Sparticuz/chromium](https://github.com/Sparticuz/chromium/tree/master) repository. 
It doesn't contain a `node_modules` folder that could be _required_.

When I use `v35` of the layer I get the following error message.

```
 "Error: Cannot find module '@sparticuz/chromium'
Require stack:
- /var/task/browsershot.js
- /var/runtime/index.mjs. [TRACE] Runtime.ImportModuleError: Error: Cannot find module '@sparticuz/chromium' Require stack: - /var/task/browsershot.js".
```

When I create a new layer using `chromium-v116.0.0-layer.zip` from the latest [Sparticuz/chromium-release](https://github.com/Sparticuz/chromium/releases/tag/v116.0.0), my script finds the chromium instance.

Comment (iwaduarte) [reply .github/workflows/layers_dispatch.yaml:34]: Oh yep. You are definitely right @stefanzweifel.  I have made a mistake. The download_url property is the url download not the zipball_url 😢.  

@vladgolubev

Comment (stefanzweifel) [reply .github/workflows/layers_dispatch.yaml:34]: @iwaduarte No problem. :) 
Thanks again for making this PR.

Comment (vladgolubev) [reply .github/workflows/layers_dispatch.yaml:34]: I've published a fix, feel free to try now!

Comment (stefanzweifel) [reply .github/workflows/layers_dispatch.yaml:34]: Works flawlessly so far. Thank y'all! ♥️

**Pull request merged**